src/firefox/remote.js

```diff
@@ -151,3 +196,10 @@ export class RemoteFirefox {
       });
   }
 }
+
+// NOTE: this fix an issue with flow and default exports (which currently
+// lose their type signatures) by explicitly declare the default export
+// signature. Reference: https://github.com/facebook/flow/issues/449
+declare function exports( // eslint-disable-line no-unused-vars
+  port: number, extraParams?: ConnectExtraParams
+): Promise<RemoteFirefox>;
```

Comment:
The issue you linked to seems to talk more about `declare module...` syntax. Is there a different issue to link to? Also, I think it would be helpful to move this underneath `connect` so that maybe it will be more obvious when we have to change the signature.